src/AppBundle/Entity/Base/PostalAddress.php

```diff
@@ -78,11 +91,31 @@
      * @Groups({"place"})
      * @ORM\Column(nullable=true)
      * @Assert\Type(type="string")
+     * @Assert\NotBlank(groups={"delivery_address"})
      * @ApiProperty(iri="https://schema.org/streetAddress")
      */
     protected $streetAddress;
 
     /**
+     * @var string Second field for street address, typically additional informations.
+     *
+     * @ORM\Column(nullable=true)
+     * @Assert\Type(type="string")
+     * @ApiProperty(iri="https://schema.org/streetAddress")
+     */
+    protected $streetAddress2;
+
+    /**
+     * @var string Phone Number, mandatory for delivery.
+     *
+     * @ORM\Column(nullable=true)
+     * @Assert\Type(type="string")
+     * @Assert\NotBlank(groups={"delivery_address"})
+     * @ApiProperty(iri="https://schema.org/streetAddress")
+     */
+    protected $phoneNumber;
```

Comment:
This should be in the `ApiUser` class IMHO. 
One customer may have several addresses (home, work, a friend's house), but the phone number is the same.